standard/operator.lua

```diff
@@ -67,11 +67,14 @@ end
 ---@param item string|number
 ---@return fun(tbl: table): any
 function Operator.property(item)
-	if string.find(item, '%.') then
-		error('Pathing not yet supported in property')
-	end
+	local pathSegments = mw.text.split(item, '.', true)
```

Comment:
Needs a branch in case item is a number, as otherwise the number is converted to string:
```
mw.logObject(mw.text.split(1, '.', true))
table#1 {
    "1",
}
```

Reply:
selected = selected[pathSegment] or selected[tonumber(pathSegment)]